test/concurrent.test.js

```diff
@@ -163,6 +165,9 @@ const runOneTest = async (ti, tj) => {
   const ei = TestEditor.create();
   const ej = TestEditor.create();
 
+  ei.syncDoc.set("data", new Y.Map())
+  ei.syncDoc.set("document", new Y.Array())
```

Comment:
Similar to comment above, can this be folded into TestEditor.create()?

Reply:
It's the same behavior explained at the collaboration suggestion.

Reply:
Agreed. (Whatever we settle on there, this should align.)